doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-provider-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='business-central-security-management-provider-con_{context}']
+
+= Security management providers
+
+In the context of security management, a realm restricts access to different application resources. Realms contain information about the users, groups, roles, and permissions. A concrete user and group management service implementation for a specific realm is called a security management provider.
```

Comment:
Realms contain information about the users, groups, roles, and permissions.

Remove 'the'.

Reply:
Done